<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvertDialog</class>
 <widget class="QDialog" name="ConvertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convert</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox_targetfile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>target file</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_targetfile">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_targetfile">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Browse</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_convert">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Convert</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
